### ISSUE TITLE
 Illegal byte sequence while install

### ISSUE THREAD

**Issue by evtuhovdo** [zbateson/MailMimeParser#61]
`
php composer.phar require zbateson/mail-mime-parser`

```
- Installing zbateson/mail-mime-parser (0.4.8): Loading from cache
    Failed to execute unzip -qq  '/Users/evtuhovdo/www/project/vendor/zbateson/mail-mime-parser/0786fef0db1349231721e3580a31f74a' -d '/Users/evtuhovdo/www/project/vendor/composer/4a5fe707'

error:  cannot create /Users/evtuhovdo/www/project/vendor/composer/4a5fe707/zbateson-MailMimeParser-b05f32a/tests/_data/emails/files/HasenundFr��sche.txt
        Illegal byte sequence

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class

```

**Comment by zbateson** [comment]
@evtuhovdo thanks for the report -- will probably just have to rename the file in the repo.

Does it then try to use ZipArchive and succeed, or does composer fail after that?

**Comment by zbateson** [comment]
Perfect, thanks again :+1:

**Comment by zbateson** [comment]
Released in 0.4.9.
